src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -322,35 +322,40 @@ event::ptr primitive_inst::execute(const std::vector<event::ptr>& events) {
 
     std::vector<event::ptr> dependencies;
     if (is_dynamic()) {
-        update_shape();
-        if (!is_valid_fusion()) {
-            auto subgraph = get_unfused_subgraph();
-
-            for (auto& d : _deps) {
-                if (!d->get_node().is_type<data>()) {
-                    subgraph->set_input_data(d->id(), d->output_memory_ptr());
+        try {
```

Comment:
Why?

Reply:
When this code block has some exception, we cannot get the detailed error message from the application. we just got the simple message such as 'std::exception'. So, I added try catch clause to print detailed error message for exception in this code block.

Reply:
If you see this issue in benchmark_app, then it can be fixes easily in infer_request_wrap.hpp by replacing 
```
    try {
        std::rethrow_exception(inferenceException);
    } catch (const std::exception& ex) {
        throw ex;
    }
```
with
```
    std::rethrow_exception(inferenceException);
```

Anyway, This kind of exception handling doesn't look good to me, so please revert this change.

Reply:
Done.